apps-rendering/src/components/orderedList.tsx

```diff
@@ -0,0 +1,42 @@
+// ----- Imports ----- //
+
+import { css } from '@emotion/react';
+import type { SerializedStyles } from '@emotion/react';
+import { remSpace } from '@guardian/src-foundations';
+import type { FC, ReactNode } from 'react';
+
+// ----- Component ----- //
+
+const styles: SerializedStyles = css`
+	list-style: none;
+	margin: ${remSpace[3]} 0;
+	padding-left: 0;
+	clear: both;
+
+	counter-reset: li;
+	> li:before {
+		content: counter(li);
+		counter-increment: li;
+		display: block;
+	}
+
+	> li {
+		padding-left: 0;
+	}
+
+	> li p:first-of-type {
+		display: block;
+	}
```

Comment:
Are you sure this rule is needed?

Reply:
Without this rule, the elements in the ordered list appear with a very less gap between them as shown in the picture below. Note in `listItem`, the `p` has `display: inline`:
```
> p:first-of-type {
	display: inline;
	padding: 0;
}
```

![image](https://user-images.githubusercontent.com/91967886/140959869-22728b80-4d99-4b11-99ce-998169773941.png)